website/docs/content/punctuation/punctuation.md

```diff
@@ -44,24 +44,24 @@ title: Punctuation and special symbols
 | Symbol       | HTML entity        |
 | :----------: | :----------------: |
 | &mdash;      | `&mdash;`          |
-
 * Use sparingly in sentences in place of a comma or colon
 * Omit the spaces on both sides of the dash
 
 <table class="dosdonts">
     <tr>
-        <th>Don't</th>
+        <th>Don’t</th>
         <th>Do</th>
     </tr>
     <tr>
         <td>
-            It makes us sound like Semrush &mdash; regardless of who the person behind the communication&nbsp;is.
+            It makes us sound like Semrush &mdash; regardless<br />of who the personbehind the communication is.
         </td>
         <td>
-            It makes us sound like Semrush&mdash;regardless of who the person behind the communication&nbsp;is.
+            It makes us sound like Semrush&mdash;regardless<br />of who the personbehind the communication is.
```

Comment:
I know about the missing space, but I'm not sure I'm able to add a quick fix to the current PR